### PR TITLE
 Adds detection for multiple notebooks

### PR DESCRIPTION
**Pull request by liviuconcioiu**

### CODE REVIEW

File: misc/statistics.php [hunk 1]
```diff
@@ -109,5 +110,6 @@ function format($str, $length)
     format($deviceTypes[8], $parsedUAs), getPercentage($deviceTypes[8], $parsedUAs),
     format($deviceTypes[9], $parsedUAs), getPercentage($deviceTypes[9], $parsedUAs),
     format($deviceTypes[10], $parsedUAs), getPercentage($deviceTypes[10], $parsedUAs),
+    format($deviceTypes[10], $parsedUAs), getPercentage($deviceTypes[13], $parsedUAs),
```
Comment (mimmi20) [inline misc/statistics.php:113]: Should this not be `format($deviceTypes[13], $parsedUAs), getPercentage($deviceTypes[13], $parsedUAs)`

**Review by mimmi20 — CHANGES_REQUESTED** [review]

**Comment by sgiehl** [comment]
@liviuconcioiu  I'm not sure if we should add a new device type notebook 🤔  Nowadays most people are using notebooks, but most of the can't be detected as such and will be shown as desktop. Starting to detect some of them as notebooks now (and only those using facebook), might give a wrong insight

**Comment by liviuconcioiu** [comment]
I see. I will move them to desktop then.

**Comment by sgiehl** [comment]
@liviuconcioiu I'll merge this to a local branch an recreate the PR with the required changes...

**Pull request merged**